src/Form/Button.cpp

```diff
@@ -150,6 +150,9 @@ Button::OnKeyCheck(unsigned key_code) const
 {
   switch (key_code) {
   case KEY_RETURN:
+  #ifdef ANDROID
```

Comment:
Bad indent

Reply:
have moved this and all the others all the way to the left.